src/bot/logic/awareness.ts

```diff
@@ -215,14 +198,61 @@ export class MatchAwarenessImpl implements MatchAwareness {
                 .getPlayers()
                 .filter((p) => p !== playerData.name && !game.areAlliedPlayers(playerData.name, p));
             const enemy = game.getPlayerData(enemyPlayers[0]);
-            this.mainRallyPoint = getPointTowardsOtherPoint(
-                game,
-                playerData.startLocation,
-                enemy.startLocation,
-                10,
-                10,
-                0,
-            );
+            // 使用findPath找到从我方startLocation到enemy.startLocation的Track路径70%位置
```

Comment:
sorry, but can we translate all of these comments to English (at least have both languages) - the codebase is in English